packages/p2p/src/utils/orders.js

```diff
@@ -0,0 +1,152 @@
+});
+
+const getOrderLabels = (is_my_ad, is_buy_ad, order_details) => {
+    const labels = {};
+
+    if (is_buy_ad) {
+        if (is_my_ad) {
+            labels.other_party_role = localize('Buyer');
+            labels.left_send_or_receive = localize('Receive');
+            labels.right_send_or_receive = localize('Send');
+            labels.payment_details = localize('Your payment details');
+            labels.contact_details = localize('Your contact details');
+            labels.instructions = localize('Your instructions');
+            labels.result_string = localize('You sold {{offered_amount}} {{offered_currency}}', {
+                offered_amount: order_details.amount_display,
+                offered_currency: order_details.account_currency,
+            });
+        } else {
+            labels.other_party_role = localize('Seller');
+            labels.left_send_or_receive = localize('Send');
+            labels.right_send_or_receive = localize('Receive');
+            labels.payment_details = localize("Seller's payment details");
+            labels.contact_details = localize("Seller's contact details");
+            labels.instructions = localize("Seller's instructions");
+            labels.result_string = localize("You've received {{offered_amount}} {{offered_currency}}", {
+                offered_amount: order_details.amount_display,
+                offered_currency: order_details.account_currency,
+            });
+        }
+    } else if (is_my_ad) {
+        labels.other_party_role = localize('Seller');
+        labels.left_send_or_receive = localize('Send');
+        labels.right_send_or_receive = localize('Receive');
+        labels.payment_details = localize("Seller's payment details");
+        labels.contact_details = localize("Seller's contact details");
+        labels.instructions = localize('Your instructions');
+        labels.result_string = localize("You've received {{offered_amount}} {{offered_currency}}", {
+            offered_amount: order_details.amount_display,
+            offered_currency: order_details.account_currency,
+        });
+    } else {
+        labels.other_party_role = localize('Buyer');
+        labels.left_send_or_receive = localize('Receive');
+        labels.right_send_or_receive = localize('Send');
+        labels.payment_details = localize('Your payment details');
+        labels.contact_details = localize('Your contact details');
+        labels.instructions = localize("Buyer's instructions");
+        labels.result_string = localize('You sold {{offered_amount}} {{offered_currency}}', {
+            offered_amount: order_details.amount_display,
+            offered_currency: order_details.account_currency,
+        });
+    }
+
+    return labels;
+};
+
+export const getExtendedOrderDetails = (order_details, loginid) => {
+    const { advert_details, advertiser_details, client_details, status, type } = order_details;
+
+    const is_buy_ad = type === 'buy';
```

Comment:
can put statuses in constants

Reply:
We're taking care of this is another PR.